packages/client/wallets/aa/src/utils/signer.ts

```diff
@@ -25,12 +25,22 @@ export async function createOwnerSigner(
     walletConfig: WalletConfig,
     crossmintService: CrossmintService
 ): Promise<SmartAccountSigner> {
-    if ((walletConfig.signer as FireblocksNCWSigner)?.type === "FIREBLOCKS_NCW") {
-        const passphrase = (walletConfig.signer as FireblocksNCWSigner).passphrase;
-        const fireblocks = await FireblocksNCWallet(user.email, crossmintService, chain, passphrase);
+    if (isFireblocksNCWSigner(walletConfig.signer)) {
+        const { passphrase } = walletConfig.signer;
+        let fireblocks;
+
+        if ("walletId" in walletConfig.signer && "deviceId" in walletConfig.signer) {
```

Comment:
if `FireblocksNCWallet` took an object, or even the full `FireblocksNCWSigner` type, u could just pass it into the function instead of having to do `"walletId" in walletConfig.signer && "deviceId" in walletConfig.signer` and the seperate if statements